cfn-resources/database-user/mongodb-atlas-databaseuser.json

```diff
@@ -140,7 +142,8 @@
       },
       "minItems": 1,
       "type": "array",
-      "uniqueItems": true
+      "uniqueItems": true,
+      "insertionOrder": false
```

Comment:
why is insertionOrder needed? doesn't it default to false?

Reply:
it defaults to true as you can see in the [docs](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/resource-type-schema.html)

Reply:
now that I think about it, the behaviour is changed with this change, because default is true and now I'm setting it to false even though I think it makes sense to be false. I can change it to set it to the default value.